include/aws/auth/credentials.h

```diff
@@ -715,6 +715,24 @@ struct aws_credentials_provider_cognito_options {
     struct aws_auth_http_system_vtable *function_table;
 };
 
+/**
+ * Configuration options for `aws_credentials_new_with_options`
+ */
+struct aws_credentials_options {
+
+    /* Value for the aws access key id field */
+    struct aws_byte_cursor access_key_id_cursor;
+    /* Value for the secret access key field */
+    struct aws_byte_cursor secret_access_key_cursor;
+    /* (Optional) security token associated with the credentials */
+    struct aws_byte_cursor session_token_cursor;
+    /* (Optional) account id associated with the credentials */
+    struct aws_byte_cursor account_id_cursor;
+    /* expiration_timepoint_seconds timepoint, in seconds since epoch, that the credentials will no longer be valid.
```

Comment:
trivial: none of the other docs copy/pasted the variable name
```suggestion
    /* Time point, in seconds since epoch, that the credentials will no longer be valid.
```